nicoRDL/dotTheory-JS
# JS-203 (JS-353) Interacting with the DOM: Responding to Events

## Related Ticket

https://codeandtheory.atlassian.net/browse/JS-353

## Description
- completed tutorial on responding to DOM events using eventListeners
- completed tutorial quizzes

## Steps to Validate
1. Open the index.html file
2. Click on the "Make it Red!" button
3. The Warning element should now have a RED background

https://github.com/nicoRDL/dotTheory-JS/assets/100089185/ee3efa19-5df6-4c1a-ad70-184390c9363b


## Risks
N/A
